src/frontends/pytorch/src/transforms/min_prim_list_construct_replacer.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "min_prim_list_construct_replacer.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/reduce_min.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+MinPrimListConstructReplacer::MinPrimListConstructReplacer() {
+    auto min_op = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
+
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto min_op = cast_fw_node(m.get_match_root(), "prim::min");
+        if (!min_op) {
+            return false;
+        }
```

Comment:
```suggestion
        auto min_op = cast_fw_node(m.get_match_root(), "prim::min");
        auto max_op = cast_fw_node(m.get_match_root(), "prim::max");
        if (!min_op && !max_op) {
            return false;
        }
```

And later to figure out if you are working with min or max you can always use `if (min_op)`